### ISSUE TITLE
Not all entity categories have unique ids

### ISSUE THREAD

**Issue by steynovich** [leeyuentuen/alfen_wallbox#25]
Adding two Alfen devices in Home Assistant is generating errors, because not all identities have unique ids. Resulting in ignored entities for subsequent devices. Not all entity categories are affected, but according my log files the following categories don't have unique ids:
* number
* switch
* select

```2023-07-24 15:27:13.989 ERROR (MainThread) [homeassistant.components.select] Platform alfen_wallbox does not generate unique IDs. ID None_lb_solar_charging_mode already exists - ignoring select.charger_wand_solar_charging_mode
2023-07-24 15:27:13.996 ERROR (MainThread) [homeassistant.components.select] Platform alfen_wallbox does not generate unique IDs. ID None_lb_solar_charging_boost already exists - ignoring select.charger_wand_solar_charging_boost
2023-07-24 15:27:14.000 ERROR (MainThread) [homeassistant.components.select] Platform alfen_wallbox does not generate unique IDs. ID None_alb_phase_connection already exists - ignoring select.charger_wand_active_load_balancing_phase_connection
2023-07-24 15:27:14.002 ERROR (MainThread) [homeassistant.components.select] Platform alfen_wallbox does not generate unique IDs. ID None_auth_mode already exists - ignoring select.charger_wand_authorization_mode
2023-07-24 15:27:14.006 ERROR (MainThread) [homeassistant.components.select] Platform alfen_wallbox does not generate unique IDs. ID None_load_balancing_mode already exists - ignoring select.charger_wand_load_balancing_mode
2023-07-24 15:27:14.009 ERROR (MainThread) [homeassistant.components.select] Platform alfen_wallbox does not generate unique IDs. ID None_lb_active_balancing_received_measurements already exists - ignoring select.charger_wand_load_balancing_received_measurements
2023-07-24 15:27:14.012 ERROR (MainThread) [homeassistant.components.select] Platform alfen_wallbox does not generate unique IDs. ID None_display_language already exists - ignoring select.charger_wand_display_language
2023-07-24 15:27:14.015 ERROR (MainThread) [homeassistant.components.select] Platform alfen_wallbox does not generate unique IDs. ID None_bo_network_1_connection_priority already exists - ignoring select.charger_wand_backoffice_network_1_connection_priority_ethernet
2023-07-24 15:27:14.018 ERROR (MainThread) [homeassistant.components.select] Platform alfen_wallbox does not generate unique IDs. ID None_bo_network_2_connection_priority already exists - ignoring select.charger_wand_backoffice_network_2_connection_priority_gprs
2023-07-24 15:27:14.022 ERROR (MainThread) [homeassistant.components.select] Platform alfen_wallbox does not generate unique IDs. ID None_socket_1_operation_mode already exists - ignoring select.charger_wand_socket_1_operation_mode
2023-07-24 15:27:14.025 ERROR (MainThread) [homeassistant.components.binary_sensor] Platform alfen_wallbox does not generate unique IDs. ID None_system_date_light_savings already exists - ignoring binary_sensor.charger_wand_system_daylight_savings
2023-07-24 15:27:14.029 ERROR (MainThread) [homeassistant.components.switch] Platform alfen_wallbox does not generate unique IDs. ID None_enable_phase_switching already exists - ignoring switch.charger_wand_enable_phase_switching
2023-07-24 15:27:14.032 ERROR (MainThread) [homeassistant.components.switch] Platform alfen_wallbox does not generate unique IDs. ID None_dp_light_auto_dim already exists - ignoring switch.charger_wand_display_light_auto_dim
2023-07-24 15:27:14.035 ERROR (MainThread) [homeassistant.components.number] Platform alfen_wallbox does not generate unique IDs. ID alb_safe_current already exists - ignoring number.charger_wand_alb_safe_current
2023-07-24 15:27:14.038 ERROR (MainThread) [homeassistant.components.number] Platform alfen_wallbox does not generate unique IDs. ID current_limit already exists - ignoring number.charger_wand_current_limit
2023-07-24 15:27:14.042 ERROR (MainThread) [homeassistant.components.number] Platform alfen_wallbox does not generate unique IDs. ID max_station_current already exists - ignoring number.charger_wand_max_station_current
2023-07-24 15:27:14.044 ERROR (MainThread) [homeassistant.components.number] Platform alfen_wallbox does not generate unique IDs. ID max_smart_meter_current already exists - ignoring number.charger_wand_max_meter_current
2023-07-24 15:27:14.049 ERROR (MainThread) [homeassistant.components.number] Platform alfen_wallbox does not generate unique IDs. ID lb_solar_charging_green_share already exists - ignoring number.charger_wand_solar_green_share
2023-07-24 15:27:14.053 ERROR (MainThread) [homeassistant.components.number] Platform alfen_wallbox does not generate unique IDs. ID lb_solar_charging_comfort_level already exists - ignoring number.charger_wand_solar_comfort_level
2023-07-24 15:27:14.058 ERROR (MainThread) [homeassistant.components.number] Platform alfen_wallbox does not generate unique IDs. ID dp_light_intensity already exists - ignoring number.charger_wand_display_light_intensity
2023-07-24 15:27:14.063 ERROR (MainThread) [homeassistant.components.number] Platform alfen_wallbox does not generate unique IDs. ID lb_max_imbalance_current already exists - ignoring number.charger_wand_max_imbalance_current_between_phases```

**Comment by leeyuentuen** [comment]
i've merged them to dev for testing before release

**Comment by steynovich** [comment]
@leeyuentuen Thanks. Yes, I can confirm it's working now. I've two charging stations with the same number (all) of entities now. And no more errors in the logs.

For the changelog we might need to add a remark that this change might break integrations, because entities are getting renamed. This will also result in having unused entities under each device. Best solution is to re-add your charging station to get rid of the unused entities.

**Comment by leeyuentuen** [comment]
i'll put them in the release not that it has a breaking changes

**Comment by leeyuentuen** [comment]
thanks for the commit, i'll close this issue. It has been released on 2.3.2 for public
